packages/common/src/api/tan-query/comments.ts

```diff
@@ -86,7 +87,7 @@ export const useUserComments = (
       if (lastPage?.length < pageSize) return undefined
       return (pages.length ?? 0) * pageSize
     },
-    queryKey: [QUERY_KEYS.userCommentList, userId],
+    queryKey: [QUERY_KEYS.userCommentList, userId, source],
```

Comment:
just double checking that you actually need separate data for the two pages

Reply:
yea they were messing each other up with the different page sizes. Unless there is another way to refetch if we change the pageSize?

Reply:
you could include pageSize in the key instead

Reply:
but either one should work

techincally, the way you have it breaks if you change the page size without changing the source, since it messes up the `getNextPageParam` math